Nanotrasen Office Ruin

BBBBAface opened the pull request:
## About the pull request
Adds a new ruin, a semi-recently abandoned Nanotrasen office building. This is my first ruin, and now that I have familiarized myself with the tools and processes involved, I will be making larger ones in the near future.

![2022 09 10-01 02 35](https://user-images.githubusercontent.com/19318747/189469885-0a827557-097a-48b3-97e6-27f559909909.png)


There is a decent amount of gear in this ruin, with the main loot item being an Origami handbook and a couple handguns, as well as two security lockers full of the default loot. However, this is all defended by a couple dozen stickmen and stickdogs, as well as Mjor the Creative.

Features restorable power

Extended list of loot:
The Art of Origami
Dronespeak Manual
Random Manual x4
Commander x2
SecTech vendor
Security Officer's Locker x2
Jukebox
Personal Ai device
Paperwork the sloth
Deep Fryer x2
Microwave x2
Refrigerator (with milk and eggs) x4
Super P.A.C.M.A.N. 
Uranium sheet x5
Worn-Out Insulated Gloves
Hundreds of sheets of paper
Genius research papers (hidden)


![image](https://user-images.githubusercontent.com/19318747/187530967-887db674-8634-4a4b-bd4d-6917f4d80abf.png)

![image](https://user-images.githubusercontent.com/19318747/187531032-e98d8a2b-6b52-4e66-a3b7-7c6f62c039f3.png)

I affirm that I have tested all of my proposed changes and that any issues found during testing have been addressed.

## Why is it good for the game?

We need more ruins, especially in space, and is an interesting one to explore. There is decent loot and a high risk of death due to many enemies. I've run through it with the starter gear from a couple of ships, and while it is difficult, it isn't impossible.

BBBBAface committed:
Add files via upload

BBBBAface committed:
Add files via upload

BBBBAface committed:
Delete datums/ruins directory

Signed-off-by: BBBBAface <nonyabeezwahx@yahoo.com>

BBBBAface committed:
Delete areas/ruins directory

Signed-off-by: BBBBAface <nonyabeezwahx@yahoo.com>

Latentish commented:
This level is good and cool. My problem with it is the lootlist. It's a bit of a lootbox, and has things that NT just probably wouldn't have like a codespeak manual or an uzi.

BBBBAface committed:
Add files via upload

added the actual map that I definitely didn't forget to upload initially

BBBBAface commented:
> This level is good and cool. My problem with it is the lootlist. It's a bit of a lootbox, and has things that NT just probably wouldn't have like a codespeak manual or an uzi.

Thanks! I'll tweak the loot a little bit

BBBBAface committed:
Add files via upload

Changed Codespeak manual to Dronespeak manual, and an Uzi into a third Commander

BBBBAface committed:
Fixed errors hopefully, added portable air scrubber

Valorium-RB commented:
A more appropriate SMG for NT would be the SABR. I also believe the Vector can be used by NT, but I'm not certain.

BBBBAface committed:
Fixed windows and power generator room

I also nerfed the loot a little bit

BBBBAface committed:
Merge branch 'shiptest-ss13:master' into master

BBBBAface committed:
final update - fixes APCs

BBBBAface committed:
Merge branch 'shiptest-ss13:master' into master

Latentish commented on code/game/area/areas/ruins/space.dm:
```diff
@@ -543,3 +543,29 @@
 	icon_state = "dk_yellow"
 	color = "#26773a88"
+
+// Nanotrasen Office
```
You could cut down on areas here or use generics that are appropriate for your planet type.

BBBBAface replied:
I'll fix this soon, anything I should change aside from the passthroughs?

Latentish reviewed (changes requested):
The massive glaring issue with this map is your areas.
You have turfs with area passthrough on top of them.
Area Passthrough must always be paired with a Turf Passthrough, and any tile that has them can't contain anything else. This otherwise causes your map to have a chance of spawning improperly placed space tiles with fucked up areas. Please fix this.

BBBBAface committed:
Fixes area issue

All good now

BBBBAface committed:
Merge branch 'shiptest-ss13:master' into master

BBBBAface committed:
Merge branch 'master' into master

Latentish commented:
Aside from cutting your areas down and not adding to the blacklist, I don't have any issues with this.

BBBBAface commented:
> Aside from cutting your areas down and not adding to the blacklist, I don't have any issues with this.

I don't use github often, my local file has this ruin on the blacklist. does this really not?

BBBBAface committed:
Merge branch 'shiptest-ss13:master' into master

BBBBAface committed:
Merge branch 'shiptest-ss13:master' into master

BBBBAface committed:
forgot a tile

Latentish commented:
> > Aside from cutting your areas down and not adding to the blacklist, I don't have any issues with this.
> 
> I don't use github often, my local file has this ruin on the blacklist. does this really not?

They don't need to be there, but it seems like it's a false add from you opening the file. Just remove that false addition. You will likely need to get a github manager like github desktop or gitkraken.

BBBBAface committed:
Merge branch 'shiptest-ss13:master' into master

BBBBAface commented:
Hey, sorry I haven't been looking at code prs for a while due to the cleanup from hurricane Ian. What exactly do you need me to do? A false add from me opening what file?
